src/platforms/javascript/common/session-replay/configuration.mdx

```diff
@@ -19,42 +18,59 @@ The following options can be configured on the root level of your browser-based
+
+### Network Details
+
+By default, Replay will capture basic information about all outgoing fetch and XHR requests in your application. This includes the URL, request and response body size, method, and status code. The intention is to limit the chance of collecting private data.
 
-You can use the Sentry SDK to set the user of a session. To associate a user identity to a session replay, refer to the <PlatformLink to="/enriching-events/identify-user">Identifying User Docs</PlatformLink>.
+To capture additional information such as request and response headers or bodies, you'll need to opt-in via `networkDetailAllowUrls`. This gives you control to add only URLs that are safe to capture bodies, avoiding any endpoints that can contain personal identifiable information (i.e., PII).
+
+Any URL matching the given pattern(s) will then be captured with additional details:
 
 ```javascript
-Sentry.setUser({ email: "jane.doe@example.com" });
+new Replay({
+  networkDetailAllowUrls: ['https://my-website.com/api'],
+});
 ```
 
-## Start and Stop Recording
+If you give us a string, we'll match any URL that contains that string. You can use a regex to handle exact or more complex matches.
 
-Replay recording starts if it's included in the `integrations` array when calling `Sentry.init` or, when `addIntegration` is called from a Sentry client instance. To stop recording, call `stop()`:
+Requests to a URL matched by the configured patterns will be enhanced with the request and response body, as well as the following default headers:
 
-```javascript
-const replay = new Sentry.Replay();
+* `content-type`
+* `content-length`
+* `accept`
 
-// Load replay when you configure Sentry SDK
-Sentry.init({
-  integrations: [replay],
-});
+If you want to capture additional headers, you'll have to configure them with the options `networkRequestHeaders` and `networkResponseHeaders`.
+
+<Note>
+  Captured bodies will be truncated to a maximum size of 150k characters. Content in
+  bodies will be PII sanitized server-side, based on object keys and values.
+</Note>
+
+### Sensitive Data in Request and Response Bodies
 
-// Or if you want to defer loading Replay
-getCurrentHub()
-  .getClient()
-  .addIntegration(replay);
+Sentry attempts to scrub certain types of sensitive data from request and response bodies, if you accidentally opt-in to a URL that includes sensitive data. This mechanism happens on our ingestion service, before data hits our disks. This is a best effort approach. Pattern matching the content with things like credit card information, social security numbers and [passwords](https://github.com/search?q=repo%3Agetsentry%2Frelay%20PASSWORD_KEY_REGEX&type=code). The best way to avoid PII getting into Sentry is to avoid adding the URL of endpoints that can contain PII.
```

Comment:
```suggestion
Sentry attempts to scrub certain types of sensitive data from request and response bodies, if you accidentally opt-in to a URL that includes this type of data. This mechanism happens on our ingestion service, before data hits our disks. This is a best effort approach which pattern-matches the content with things like credit card information, social security numbers, and [passwords](https://github.com/search?q=repo%3Agetsentry%2Frelay%20PASSWORD_KEY_REGEX&type=code). The best way to avoid getting PII into Sentry is by not adding URLs of endpoints that may contain PII.
```